front/components/assistant_builder/types.ts

```diff
@@ -217,6 +227,17 @@ export function getDefaultProcessActionConfiguration() {
   } satisfies AssistantBuilderActionConfiguration;
 }
 
+export function getDefaultWebsearchActionConfiguration() {
```

Comment:
Not a big fan of the satisfies here, it can be achieve by setting a return type.